src/components/PackPage/SoloPostView/PostMaker.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect, useContext, useRef } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import axios from 'axios';
```

Comment:
since you are not making an axios request on this page, do we need this? Or is this here for future posting?